repository/s.json

```diff
@@ -5612,6 +5601,18 @@
 				}
 			]
 		},
+		{
+			"name": "Swift Next",
```

Comment:
A non-official drive-by review but I think the syntax package should just be called by the Language name and not include a "branding" words like "next".

Reply:
Apparently this is not the case for the most useful swift package for now, as it had started with the "Swift for fucking sublime" and as I believe was changed to "Decent swift" only by Apple request they released lsp support and promoted such plugin as the default option for ST. 

Although I'm seeing no problem in renaming it by a reviewer request

Reply:
At least when the package is just a syntax package then I don't see a reason to have extra words in there. That can age very quickly.

This package also includes completions which I'm not sure is a good idea since those can be quite opinionated (even if just in terms of formatting).

You can wait for the channel owners to review it but those are my initial thoughts.

Reply:
Don't get me wrong, I didn't want it sounds like "keep out of my business". It's just the review rules are both so opaque and inconsistent for me, that I'm trying to avoid any unnecessary moves just in case. 

I see the reason in your point about the naming and snippets issues though, it's just the source of them are opaque for me as well, is it an established yet implicit practice in here, is it your personal recommendation or some another option? Whether those suggestions contradicts with @braver opinion or not?

Reply:
Let's wait for braver's review but after observing reviews in this repo for a long time, I'm fairly sure that it's not just my personal opinion.

Reply:
Package has been renamed, it looks kinda wired though, like I'm trying to replace an existing package with the different repo.

Reply:
Replacing packages normally goes through a process where the original author is notified and gets 2 weeks or so to respond.